main.py

```diff
@@ -1,22 +1,23 @@
-from fastapi import Depends, FastAPI
-from fastapi.security import OAuth2PasswordBearer
-
 from auth.management import get_management_token
-from auth.validator import verify_jwt
-from schemas.user import User
+from auth.validator import get_current_user
+from fastapi import Depends, FastAPI
+from routers import user
 
 app = FastAPI()
-oauth2_scheme = OAuth2PasswordBearer(tokenUrl="token")
-
 
-def get_current_user(token: str = Depends(oauth2_scheme)) -> User:
-    access_token = verify_jwt(token)
-    return User(access_token=access_token)
 
 @app.get("/")
 def public_route():
     return {"message": "Public route"}
 
+
 @app.get("/private")
```

Comment:
Can we remove this dummy route now? I don't like that we're returning a management token here, especially now that we're trying to automatically deploy the app.